src/main/java/Deadline.java

```diff
@@ -0,0 +1,31 @@
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+public class Deadline extends Task {
+    private LocalDate byDate;
+
```

Comment:
Also a good idea to provide Java Documentation for constructors, especially since you have exception handling logic that makes this method non-trivial 😄